dbt/models/qc/schema.yml

```diff
@@ -916,6 +892,64 @@ models:
     meta:
       export_name: Class does not equal LUC
 
+  - name: qc.vw_report_town_close_res_parcels_not_set_to_cost_approach
```

Comment:
I'm realizing now that this name might be clearer as `qc.vw_report_town_close_res_parcels_set_to_cost_approach` (as opposed to `not_set_to_cost_approach`), since that more accurately describes what is being queried for... what do you think?

Reply:
See my comment above for an explanation of why "res parcels *not* set to cost approach" is actually correct 😅